docs-v2/reference/cli.mdx

```diff
@@ -13,7 +13,19 @@ npm install nango -g
 
 In the folder where you want your integration folder (e.g. root of your project), run:
 ```bash
-nango init # Creates `./nango-integrations` with initial config
+nango init
+```
+
+This creates the `./nango-integrations` folder with some initial configuration and an example
+sync script. The `nango-integrations` directory looks like this:
+```
+nango-integrations/
+├── .env
+├── nango.yaml
+├── models.ts
+└── demo-github-integration
+    └── syncs/
+        └── github-issue-example.ts
```

Comment:
I would indicate with a comment that this is is matching the script name in the `nango.yaml`.

Reply:
Done, thanks.